docs/en/Running-Hadoop-MapReduce-on-Alluxio.md

```diff
@@ -14,56 +14,73 @@ run your MapReduce programs with files stored on Alluxio.
 
 ## Initial Setup
 
-The prerequisite for this part is that you have [Java](Java-Setup.html). We also assume that you
-have set up Alluxio and Hadoop in accordance to these guides
+The prerequisite for this guide includes
+
+- You have [Java](Java-Setup.html). 
+- You have set up an Alluxio cluster in accordance to these guides
 [Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
-In order to run some simple map-reduce examples, we also recommend you download the
-[map-reduce examples jar](http://mvnrepository.com/artifact/org.apache.hadoop/hadoop-mapreduce-examples/2.4.1) based on your hadoop version,
+- In order to run some simple map-reduce examples, we also recommend you download the
+[map-reduce examples jar](http://mvnrepository.com/artifact/org.apache.hadoop/hadoop-mapreduce-examples) based on your hadoop version,
 or if you are using Hadoop 1, this [examples jar](http://mvnrepository.com/artifact/org.apache.hadoop/hadoop-examples/1.2.1).
 
-## Compiling the Alluxio Client
+## Prepare the Alluxio client jar
+
+For the MapReduce applications to communicate with Alluxio service, it is required to have the Alluxio client jar on their classpaths. We recommend you to download this client jar from Alluxio [download page](http://www.alluxio.org/download). 
```

Comment:
done. just curious: is this requirement (100-char linewidth for md files) documented any where, as it is new to me for md files.